server/xpub/entities/manuscript/resolvers.js

```diff
@@ -43,73 +43,68 @@ const resolvers = {
 
       return manuscripts[0]
     },
-    async manuscript(_, { id }) {
-      return Manuscript.find(id)
+    async manuscript(_, { id }, { user }) {
+      return Manuscript.find(id, user)
     },
-    async manuscripts() {
-      return Manuscript.all()
+    async manuscripts(_, vars, { user }) {
+      return Manuscript.all(user)
     },
   },
 
   Mutation: {
     async createSubmission(_, vars, ctx) {
+      if (!ctx.user) {
+        throw new Error('Not logged in')
+      }
       const manuscript = Manuscript.new()
       manuscript.createdBy = ctx.user
       return Manuscript.save(manuscript)
     },
 
-    async deleteManuscript(_, { id }, ctx) {
-      const originalManuscript = await Manuscript.find(id)
-      Manuscript.checkPermission(originalManuscript, ctx.user)
+    // TODO restrict this in production
+    async deleteManuscript(_, { id }, { user }) {
+      await Manuscript.find(id, user)
 
       await Manuscript.delete(id)
       return id
     },
 
-    async updateSubmission(_, { data }, ctx) {
-      const originalManuscript = await Manuscript.find(data.id)
-      Manuscript.checkPermission(originalManuscript, ctx.user)
+    async updateSubmission(_, { data }, { user }) {
```

Comment:
If you used destructuring here for user, why don't you use it everywhere else as well?